web/src/stream_events.js

```diff
@@ -109,6 +109,9 @@ export function update_property(stream_id, property, value, other_values) {
         case "can_remove_subscribers_group":
             stream_settings_ui.update_can_remove_subscribers_group(sub, value);
             break;
+        case "can_administer_channel_group":
+            stream_settings_ui.update_can_administer_channel_group(sub, value);
+            break;
```

Comment:
This probably can be rewritten to use the `group_permission_settings.ts` / `realm.server_supported_permission_settings` to process these generically, rather than having similar little functions for each one. Small code cleanup compared to the backend ones though.